packages/govuk-frontend/src/govuk/objects/_form-group.scss

```diff
@@ -3,7 +3,7 @@
 @include govuk-exports("govuk/objects/form-group") {
   .govuk-form-group {
     @include govuk-clearfix;
-    @include govuk-responsive-margin(6, "bottom");
+    margin-bottom: govuk-spacing(6);
```

Comment:
It's quite typical to see `.govuk-form-group`, `.govuk-button-group` and `.govuk-button` together

This change leaves two of those with a "responsive" margin still

Can the vertical rhythm drift off-beat for a little while? 🥁